automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeGceSpec.scala

```diff
@@ -45,7 +44,7 @@ class RuntimeGceSpec
 
   "should create a GCE instance in a non-default zone" in { project =>
     val runtimeName = randomClusterName
-    val diskName = genDiskName.sample.get
+    val diskName = randomDiskName()
```

Comment:
hmm.... this has to be a bug, its used as a function here and a string in other places such as `RuntimeCreationDiskSpec`?